astraceroute.c

```diff
@@ -65,8 +79,8 @@ struct proto_ops {
 	size_t min_len_tcp, min_len_icmp;
 	int (*check)(uint8_t *packet, size_t len, int ttl, int id,
 		     const struct sockaddr *src);
-	void (*handler)(uint8_t *packet, size_t len, int dns_resolv,
-			int latitude);
+	void (*handler)(uint8_t *packet, size_t len, int do_dns_resolution,
+			int do_geo_lookup);
```

Comment:
Change `do_dns_resolution` and `do_geo_lookup` to `bool` type while at it.

Reply:
Done.